Tools/Scripts/webkitdirs.pm

```diff
@@ -614,7 +614,7 @@ sub determineXcodeDestination
         $destination .= 'platform=visionOS Simulator';
     } else {
         $destination .= 'platform=macOS';
-        $destination .= ',arch=' . $architectures[0];
+        $destination .= ',arch=' . $architectures[0] if !$generic;
```

Comment:
I think that `unless` would be slightly nicer here (probably not worth actually changing unless making a more important change in nearby code).

Reply:
Rolling this in to https://github.com/WebKit/WebKit/pull/23200.